src/main/java/dev/slimevr/vr/processor/skeleton/LegTweakBuffer.java

```diff
@@ -817,17 +818,12 @@ private void findForceVectors(Vector3f leftFootForce, Vector3f rightFootForce) {
 	private boolean detectOutsideForces(Vector3f f1, Vector3f f2) {
 		Vector3f force = GRAVITY.add(f1).add(f2);
 		Vector3f error = centerOfMassAcceleration.subtract(force);
-		return error.length() > FORCE_ERROR_TOLLERANCE;
+		return error.lengthSquared() > FastMath.sqr(FORCE_ERROR_TOLLERANCE);
```

Comment:
`FORCE_ERROR_TOLLERANCE` can be squared in the constant to get rid of this one multiplication operation (`FastMath.sqr`), this is absolutely a micro optimization, even moreso than the last, so it's fine if it gets merged with this